simulation_parameters/Constants.cs

```diff
@@ -1314,6 +1314,10 @@ public static class Constants
     public const float VENT_ERUPTION_HYDROGEN_SULFIDE_INCREASE = 0.00004f;
     public const float VENT_ERUPTION_CARBON_DIOXIDE_INCREASE = 0.3f;
 
+    public const float OXYGEN_THRESHOLD = 0.07f;
+    public const float OXYGEN_PATCHES_THRESHOLD = 0.7f;
+    public const int GLOBAL_GLACIATION_CHANCE = 50;
```

Comment:
I think it is much cleaner if all of our changes are in the range of 0-1 and not use implicit percentages.